attributes/geo:proj/README.md

```diff
@@ -0,0 +1,237 @@
+# Projection Attribute Extension for Zarr
+
+- **Extension Name**: Projection Attribute Extension
+- **Version**: 1.0.0
+- **Extension Type**: Attribute
+- **Status**: Proposed
+- **Owners**: @emmanuelmathot
+
+## Description
+
+This specification defines a JSON object that encodes coordinate reference system (CRS) information for geospatial data. Additionally, this specification defines a convention in which this object is stored under the `"geo:proj"` key in the attributes of Zarr groups or arrays.
+
+**Recommended usage**: Define `geo:proj` at the **group level** to apply CRS information to all arrays within that group. This matches the common geospatial pattern of storing multiple arrays with the same coordinates in a single group. Array-level definitions are supported for override cases but are less common.
+
+## Motivation
+
+- Provides simple, standardized CRS encoding without complex nested structures
+- Compatible with existing geospatial tools (GDAL, rasterio, pyproj)
+- Based on the proven STAC Projection Extension model
+
+## Inheritance Model
+
+The `geo:proj` attribute follows a simple group-to-array inheritance model that should be understood first:
+
+### Inheritance Rules
+
+1. **Group-level definition** (recommended): When `geo:proj` is defined at the group level, it applies to all arrays within that group
+2. **Array-level override**: An array can completely override the group's `geo:proj` attribute with its own definition
+3. **Complete replacement only**: Partial inheritance (overriding only some fields while inheriting others) is not allowed
+4. **No cascading**: Inheritance only applies from a group directly to its immediate array members, not through nested groups
+
+Most use cases will use group-level definitions without array overrides.
+
+## Specification
+
+The `geo:proj` attribute can be added to Zarr arrays or groups to define projection information.
+
+<!-- GENERATED_SCHEMA_DOCS_START -->
+**`geo:proj` Properties**
+
+|   |Type|Description|Required|
+|---|---|---|---|
+|**version**|`string`|Projection metadata version| &#10003; Yes|
+|**code**|`["string", "null"]`|Authority:code identifier (e.g., EPSG:4326)|No|
+|**wkt2**|`["string", "null"]`|WKT2 (ISO 19162) CRS representation|No|
+|**projjson**|`any`|PROJJSON CRS representation|No|
+|**bbox**|`number` `[]`|Bounding box in CRS coordinates|No|
+|**transform**|`number` `[]`|Affine transformation coefficients|No|
```

Comment:
```suggestion
|**affine**|`number` `[]`|Affine transformation coefficients|No|
```

Could we use affine instead of transform since that's more specific and leaves an option for other transforms to be added?

Reply:
Alternatively something like

```
"transform": {"name": "affine", "configuration": {...}}
```

so it is clear that only one transform is allowed?